docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+services:
+  mysql:
+    # Always specify what version, "latest" causes issues later on
+    image: mysql:9.1
+    container_name: dev_db
+    environment:
+      MYSQL_USER: ${MYSQLDB_USER}
+      MYSQL_PASSWORD: ${MYSQLDB_PASSWORD}
+      MYSQL_ROOT_PASSWORD: ${MYSQLDB_PASSWORD_ROOT}
+      MYSQL_DATABASE: ${MYSQLDB_NAME}
+    networks:
+      - net
+    ports:
+      - "${MYSQLDB_PORT}:3306"
```

Comment:
should this be

> ${MYSQLDB_PORT}:${MYSQLDB_PORT}

Reply:
Nope, since the one on teh right is the internal port which remains unchanged, the left is teh one we use to access it from outside the container

Reply:
ah perfect thanks for clearing it up